content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,70 @@ weight: 20
+* A **resource type** is the name used in the URL (for example: `pods`, `namespaces`, `services`);
+* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**;
+* A set of instances of a resource type is known as a **collection**;
+* A single instance of the resource type is called an **object**, or sometimes a **resource**.
 
-Kubernetes generally leverages standard RESTful terminology to describe the API concepts:
+{{< note >}}
+In URLs, Kubernetes types such as `pod` and `configmap` are **lowercase**. This is different from
+how Kubernetes conventionally renders those types elsewhere, such as the `.kind` field of an
+object, where the type is typically written in **UpperCamelCase**.
 
-* A **resource type** is the name used in the URL (`pods`, `namespaces`, `services`)
-* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**
-* A list of instances of a resource type is known as a **collection**
-* A single instance of the resource type is called a **resource**
+For example: `"kind": "ConfigMap"`.
+{{< /note >}}
 
-All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource type will be deleted when its namespace is deleted and access to that resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
+All resource types are either scoped by the cluster (`/apis/<GROUP>/<VERSION>/*`) or to a namespace (`/apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/*`). A namespace-scoped resource type will be deleted when its namespace is being deleted, and access to a namespaced resource type is controlled by authorization checks on the namespace scope. The following paths are used to retrieve collections and resources:
 
 * Cluster-scoped resources:
 
-  * `GET /apis/GROUP/VERSION/RESOURCETYPE` - return the collection of resources of the resource type
-  * `GET /apis/GROUP/VERSION/RESOURCETYPE/NAME` - return the resource with NAME under the resource type
+  * `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>` - return the collection of resources of the resource type
+  * `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>/<NAME` - return the resource with NAME under the resource type
```

Comment:
It is okay to add additional decorators if that improves the readability. However, as shown by line 42, it is much easier to introduce typos when such decorators are added, i.e. there is a missing `>` at the end.

Reply:
I've omitted this change from the latest revision.